src/MusicStore/Controllers/ShoppingCartController.cs

```diff
@@ -55,15 +57,14 @@ public class ShoppingCartController : Controller
         [HttpPost]
         public async Task<IActionResult> RemoveFromCart(int id, CancellationToken requestAborted)
         {
-            var formParameters = await Context.Request.ReadFormAsync();
-            var requestVerification = formParameters["RequestVerificationToken"];
-            string cookieToken = null;
-            string formToken = null;
+            var cookieToken = string.Empty;
+            string formToken = string.Empty;
```

Comment:
I included this in https://github.com/aspnet/MusicStore/commit/1b502cbee7b86709a0b9a0afd4de4f3c97db52ae

Thanls